starlette/middleware/cors.py

```diff
@@ -48,15 +49,16 @@ def __init__(
                 "Access-Control-Max-Age": str(max_age),
             }
         )
-        if allow_headers and "*" not in allow_headers:
+        allow_headers = SAFELISTED_HEADERS | set([h.lower for h in allow_headers])
+        if allow_headers and "*" not in allowed_headers:
```

Comment:
Typo 😄 

```python
        if allow_headers and "*" not in allow_headers:
```

Not sure how this didn't get caught by CI…? It fails on master though: https://travis-ci.org/github/encode/starlette/jobs/667768111

Reply:
Thanks @florimondmanca! Fixed in https://github.com/encode/starlette/commit/c294ba6c0641b8a79204ac58919ab07d676b2920